src/views/datasets/vaccine-gap/dashboard.vue

```diff
@@ -202,6 +202,45 @@
     </template>
   </DashboardCard>
 
+  <DashboardCard width="full" :height="2">
+    <template #subtitle>Data Notes</template>
+    <template #content>
+      <div class="px-4 content">
+        <p style="font-size: 0.75rem">
```

Comment:
Perhaps add a class style to `main.scss` instead to centralize this. 

IN provident, we have
```js
// between size 6 and 7
.is-size-6-7 {
  font-size: 0.825rem !important;
}
```

Reply:
Turns out `0.75 rem` is `is-size-7`. So `class="is-size-7"`